packages/esm-patient-registration-app/src/patient-registration/section/patient-relationships/relationships.resource.tsx

```diff
@@ -0,0 +1,70 @@
+import { openmrsFetch } from '@openmrs/esm-framework';
+import { FormValues } from '../../patient-registration-types';
+
+const customRepresentation =
+  'custom:(display,uuid,' +
+  'personA:(uuid,display,person:(age,display,birthdate,uuid)),' +
+  'personB:(uuid,display,person:(age,display,birthdate,uuid)),' +
+  'relationshipType:(uuid,display,description,aIsToB,bIsToA))';
+
+export async function getPatientRelationships(patientUuid: string): Promise<FormValues['relationships']> {
+  const response = await openmrsFetch<RelationshipsResponse>(
+    `/ws/rest/v1/relationship?v=${customRepresentation}&person=${patientUuid}`,
+  );
+  if (response.ok) {
+    const relationships = response?.data?.results;
+    return relationships.map((r) =>
+      r.personA.uuid === patientUuid
+        ? {
+            relatedPersonName: r.personB.person.display,
+            relatedPersonUuid: r.personB.person.uuid,
+            relation: r.relationshipType.bIsToA,
+            relationshipType: `${r.relationshipType.uuid}/bIsToA`,
+            action: 'UPDATE',
```

Comment:
This is semantically very strange—your data-fetcher knows wayyyy too much about the context into which the data is going. But as long as no one ever tries to reuse this it will be fine. If you can think of a not-so-bad way to have `action` not appended until the data reaches the component, I think that would be nice.